pybamm/models/submodels/electrolyte/stefan_maxwell/conductivity/surface_potential_form/leading_capacitance_stefan_maxwell_conductivity.py

```diff
@@ -0,0 +1,47 @@
+#
+# Class for full surface form electrolyte conductivity employing stefan-maxwell
```

Comment:
leading order

Reply:
Thank you for flagging these inconsistensies in the doc strings. I'll check again through each model